format_checker/pr_checker.py

```diff
@@ -138,6 +140,12 @@ def check_notes(filename, row, i, log):
     if not pr_data["Notes"].fullmatch(row["Notes"]):
         log_std_error(filename, log, i, row, "Notes")
 
+def check_forked_project(filename, row, i, log):
+    with open("format_checker/forked-projects.json", "r") as f:
+        projects = json.load(f)
+    proj_url = row["Project URL"]
+    if proj_url not in projects or (proj_url in projects and projects[proj_url] == "forked"):
+        log_std_error(filename, log, i, row, "Project URL")
```

Comment:
Can you provide some descriptive message for why this row fails and what one should do to correct it (modify `forked-project.json`)?